src/main/java/com/kakaotech/team14backend/inner/point/model/GetPointPolicy.java

```diff
@@ -2,23 +2,32 @@
 
 public enum GetPointPolicy {
 
-  GET_500_WHEN_LIKE_UP("게시물 좋아요 수 50개 이상",500L),
-  GET_200_WHEN_LIKE_UP("게시물 좋아요 수 20개 이상",200L),
-  GET_100_WHEN_LINK_SNS("SNS 계정 연결",100L),
-  GET_100_WHEN_SIGN_UP("최초 회원가입",500L),
-  GET_100_REPEAT("자동 추가",100L),
-  GET_20_WHEN_LIKE_UP("좋아요 누를 시",20L),
-
-  GIVE_300_WHEN_UPLOAD("사진 업로드",300L);
+  GET_500_WHEN_LIKE_UP("게시물 좋아요 수 50개 이상", 500L),
```

Comment:
구현을 많이 드러내는 이름인 것 같은데 살짝 가리면 어떨까요?
이렇게하면 나중에 좋아요 50개 이상에 부여하는 포인트가 바뀌더라도 코드 수정이 줄어들 것 같습니다.

```suggestion
  LIKE_UP_50("게시물 좋아요 수 50개 이상", 500L),
```